static/js/components/email/constants.js

```diff
@@ -0,0 +1,2 @@
+export const SEARCH_EMAIL_TYPE = 'searchResultEmail';
+export const COURSE_EMAIL_TYPE = 'courseTeamEmail';
```

Comment:
Generally we do constants like `SEARCH_RESULT_EMAIL`, is there a strong reason for camelCase here?